File: problems/even_ones/submissions/accepted/even_ones.cpp
#include <iostream>

using namespace std;

int main(int argc, char const *argv[])
{
    int number;
    cin >> number;
    if (number % 2 == 0 && number != 2)
        cout << "YES" << endl;
    else
        cout << "NO" << endl;

    return 0;
}

File: problems/even_ones/submissions/accepted/even_ones.java
import java.util.Scanner;

public class even_ones {
    public static void main(String[] args) {
        // INPUT
        Scanner sc = new Scanner(System.in);
        int number = sc.nextInt();

        if (number % 2 == 0 && number != 2)
            System.out.println("YES");
        else
            System.out.println("NO");

    }
}